.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: ruby
+
+rvm:
+  - 2.6.5
```

Comment:
I guessed ruby-2.7 wasn't added here as the test probably didn't pass with ruby-2.7. Is this guess correct?
In that case, adding the following TODO comment will help the maintainer:

```
# We will be able to add 2.7.x here once getting new yard version including the patch for the following issue.
# https://github.com/lsegal/yard/pull/1296
```

Reply:
Nope, just didn't use 2.7. Added 2.5, 2.6 and 2.7 to the CI, all passing.